ambari-server/src/main/java/org/apache/ambari/server/controller/internal/StackServiceResourceProvider.java

```diff
@@ -277,9 +277,7 @@ private StackServiceRequest getRequest(Map<String, Object> properties) {
     return new StackServiceRequest(
         (String) properties.get(STACK_NAME_PROPERTY_ID),
         (String) properties.get(STACK_VERSION_PROPERTY_ID),
-        (String) properties.get(SERVICE_NAME_PROPERTY_ID),
-        (String) properties.get(CREDENTIAL_STORE_SUPPORTED),
-        (String) properties.get(CREDENTIAL_STORE_ENABLED));
+        (String) properties.get(SERVICE_NAME_PROPERTY_ID));
```

Comment:
Either the constructor should properly use the credential store supported and enabled; or the relevant _set_ methods should be called.